### DISCUSSION TITLE
Request for documentation: scene setup for export

### DISCUSSION THREAD

**Discussion by Allottho** [Autodesk/maya-usd#456] [Q&A]
As an artist, who needs to use USD by necessity and asap, I'm having trouble understanding how I can actually use the materials in the scene for export in any meaningful way. So far I managed to export some of the lambert, blinn and phong materials, but results are pretty uninspiring, as is probably expected. Also discovered, that tga and dds file formats are not supported, and only textures I was able to see were PNGs.
The main question is, how do I use the tools available at the moment to export something with PBR materials? (albedo-spec-roughness-metalness). So far, it's not clear.
Do you have some sort of design document for USD creation workflow we can use as reference for scene preparation?

I'd like to see some consolidated knowledge base, where I can find some information on pre-requisites needed to build a proper working USD scene, the material and lighting attributes I need to look into to get the result which is corresponding with current workflows. The page in the docs section provides a valuable info on what is currently supported, but I'd be happy to see more information on this.

It looks like Arnold USD plugin should have all I need, but it doesn't have binary releases, so it's not available for the end-user at the moment.
The question is, is it possible to achieve PBR workflow with what mayaUsdPlugin offers at the moment? Or is it a platform to build a render-specific interface which will translate scene data for specific render to the USD? 
Basically, should the artists (not render developers) even use this plugin at all or at this stage, or should they wait for render-specific version?

Thank you!
